community/portworx/templates/_helpers.tpl

```diff
@@ -20,6 +20,14 @@ productVersion: {{ .Values.imageVersion }}
 {{$version := .Capabilities.KubeVersion.GitVersion | regexFind "^v\\d+\\.\\d+\\.\\d+"}}{{$version}}
 {{- end -}}
 
+{{- define "px.getKubectlVersion" -}}
```

Comment:
Should we use the same function as the generic helm chart for consistency?
https://github.com/portworx/helm/blob/master/charts/portworx/templates/_helpers.tpl#L28

Reply:
Sounds good, right now there are many differences in charts, a little painful to maintain.

Reply:
Actually did a test, due to the kubectl image is from different docker repo, with that tag, the image is not found. I would keep as is to be safe.

Reply:
Got it. It is okay for now, but we might have issues with this in future. One such instance would be when APIs deprecate, then the newer kubectl versions may not recognize or worse try to manage objects considering the api server is updated too, which might not be the case. I believe we are not running any complicated commands using the kubectl in the hooks, so may it should be fine.